jekyll/_cci2/language-ruby.md

```diff
@@ -55,37 +58,42 @@ jobs:
     steps:
       - checkout
 
+      # Which version of bundler?
+      - run:
+          name: Which bundler?
+          command: bundle -v
+
       # Restore bundle cache
-      - type: cache-restore
-        name: Restore bundle cache
-        key: rails-demo-bundle-{{ checksum "Gemfile.lock" }}
+      - restore_cache:
+          keys:
+            - rails-demo-bundle-v2-{{ checksum "Gemfile.lock" }}
+            - rails-demo-bundle-v2-
 
       - run:
           name: Bundle Install
-          command: bundle install --path vendor/bundle
+          command: bundle check || bundle install
 
       # Store bundle cache
-      - type: cache-save
-        name: Store bundle cache
-        key: rails-demo-bundle-{{ checksum "Gemfile.lock" }}
-        paths:
-          - vendor/bundle
+      - save_cache:
+          key: rails-demo-bundle-v2-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
 
       # Only necessary if app uses webpacker or yarn in some other way
-      - type: cache-restore
-        name: Restore yarn cache
-        key: rails-demo-yarn-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          keys:
+            - rails-demo-yarn-{{ checksum "yarn.lock" }}
+            - rails-demo-yarn-
 
       - run:
           name: Yarn Install
           command: yarn install
 
       # Store yarn / webpacker cache
-      - type: cache-save
-        name: Store yarn cache
-        key: rails-demo-yarn-{{ checksum "yarn.lock" }}
-        paths:
-          - ~/.yarn-cache
+      - save_cache:
+          key: rails-demo-yarn-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.yarn-cache
```

Comment:
Might want to wait for https://github.com/CircleCI-Public/circleci-demo-ruby-rails/pull/12 to land and update this too.